digdag-docs/src/releases/release-0.10.0.rst

```diff
@@ -0,0 +1,66 @@
+Release 0.10.0
+==============
+
+General Changes
+---------------
+* Command Executor SPI has been updated and AWS ECS Command Executor has been introduced. [#835 #1127]
+* Java11 has been supported. (experimental) [#1256 #1361]
+* Graaljs has been supported for ConfigEvalEngine and performance dramatically improved. [#1256 #1361]
+* AccessController SPI has been introduced.  (experimental) [#936]
+* Metrics improvements. (experimental) [#1146]
+* Authentication configuration was changed. [#1324]
+* Abolished JWT authentication. [#1324]
+* Some API have been abolished and changed. [#1109 #1110 #1114 #1115 #1224 #1288 #1398 #1401]
+* `_parallel` option supports a number to control number of parallels. [#1413]
+* The maximum number of tasks, attempts are configurable. [#1430]
```

Comment:
[minor] `The maximum numbers of tasks and attempts are configurable` might be better?

Reply:
Thanks, I will fix it.